app/graphql/resolvers/setting_resolver.rb

```diff
@@ -0,0 +1,14 @@
+module Resolvers
+  class SettingResolver < BaseResolver
+    type Types::Setting, null: false
+
+    argument :id, String, 'Global ID for Record', required: true
```

Comment:
If we allow update by name, could we also allow it here?

Reply:
This is a global id, which should be composed from the setting name.

Reply:
I never got the appeal of global ids, especially in this context. Global id is indeed composed of several parts, one of them is name, one is a string representing class of the resource and then the whole thing is encoded as base64. Given a name uniquely identifies a setting, could we allow just using the name directly?

Reply:
I've got no idea :( it might break some compatibility somewhere down the line.
I think that the global id is supposed to identify the resource globally and should be enough to state what resource we are talking about. The name of the setting doesn't seem to be enough for sure, there might be other resources that would want to use names as identifiers. Also it would require special treatment in the resolve mechanisms.

I do not feel brave enough to implement such a thing, even tho I do not like our implementation of global id more than you do :shrug:

Reply:
Alright then